torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15449,6 +15449,7 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            aliases=('rms_norm',),
            ref=reference_rms_norm,
            dtypes=floating_and_complex_types_and(torch.half, torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
```

Comment:
regressing dtype coverage for cuda is not desirable, can you fall back for this?